src/AbstractHyperrectangle.jl

```diff
@@ -400,18 +400,18 @@ Then ``x ∈ H`` iff ``|c_i - x_i| ≤ r_i`` for all ``i=1,…,n``.
 function ∈(x::AbstractVector{N},
            H::AbstractHyperrectangle{N})::Bool where {N<:Real}
     @assert length(x) == dim(H)
-    for i in eachindex(x)
-        if abs(center(H)[i] - x[i]) > radius_hyperrectangle(H, i)
+    c = center(H)
```

Comment:
This requires to compute the full `SymmetricIntervalHull` even if you can find a counterexample in the first dimension.